package.json

```diff
@@ -33,9 +33,9 @@
     "content-disposition": "0.5.3",
     "content-type": "~1.0.4",
     "cookie": "0.4.0",
-    "cookie-signature": "1.0.6",
+    "cookie-signature": "1.1.0",
     "debug": "2.6.9",
-    "depd": "~1.1.2",
+    "depd": "~2.0.0",
```

Comment:
We want to make sure this doesn't cause duplicate `depd` stances within the install tree. I haven't checked if any of our other sub deps use `depd`, but you'll want to check that out, and then "bubble up" the dependency updated into this repo so we keep our install tree small.

Reply:
😅 Yes! You are right. Let's move it for 5.0

```
├─┬ cookie-session@1.3.3
│ ├─┬ cookies@0.7.3
│ │ ├── depd@1.1.2
```

```
├─┬ multiparty@4.2.1
│ ├─┬ fd-slicer@1.1.0
│ │ └── pend@1.2.0
│ ├─┬ http-errors@1.7.3
│ │ ├── depd@1.1.2
```

```
├─┬ send@0.17.1
│ ├── depd@1.1.2
│ ├── destroy@1.0.4
```